mlflow/cli.py

```diff
@@ -279,7 +279,15 @@ def ui(backend_store_uri, default_artifact_root, artifacts_destination, port, ho
     # TODO: We eventually want to disable the write path in this version of the server.
     try:
         _run_server(
-            backend_store_uri, default_artifact_root, artifacts_destination, host, port, None, 1
+            backend_store_uri,
+            default_artifact_root,
+            "false",
+            "false",
```

Comment:
```suggestion
            False,
            False,
```

Should these be `False`?

Reply:
It definitely should. I forgot about how Python will return a bool validation on a string of "true" or "false" the same as the bool operator.